servers/partner-ci/9.0/contrail.yaml

```diff
@@ -26,9 +26,10 @@
             VSRX_ORIGINAL_IMAGE_PATH=/storage/contrail/vSRX.400.img;
             VSRX_TARGET_IMAGE_PATH=/var/lib/libvirt/images/vSRX.400.img;
             VSRX_TEMPLATE_PATH=/storage/contrail/vSRX_400.template;
+            FORCE_DISABLE_UPDATES=true;
 - project:
     name: 'predefined_parameters'
-    fuel-release: '9.1'
+    fuel-release: '9.0'
```

Comment:
it's ok. but we have to take into account that 9.0 version is used for 9.0 and 9.1